libs/common-services/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -111,6 +111,10 @@ public static String getJourney(JourneyRequest request) {
         return request.getJourney();
     }
 
+    public static String getContext(JourneyRequest request) {
+        return request.getContext();
+    }
```

Comment:
I know this is following a pattern, but I'm not a big fan of it. This method is just a wrapper for the same method. I'd be tempted to just call `request.getContext()` when we need it, rather than using this helper.